Source/UnitTests/Core/IOS/ES/TestBinaryData.h

```diff
@@ -8,8 +8,170 @@
 #include <cstddef>
 #include "Common/CommonTypes.h"
 
-extern const std::array<u8, 520> soup01_tmd;
-extern const std::array<u8, 108> soup01_tmd_view;
+constexpr std::array<u8, 520> soup01_tmd = {{
```

Comment:
Just wondering, what warning does this fix?

Reply:
`extern const` is not equivalent to the actual storage type (which is `constexpr`) and apparently [extern constexpr is not possible](https://stackoverflow.com/questions/30208685/how-to-declare-constexpr-extern)

Reply:
Perhaps the existing code could be used, msvc just requires another flag: https://docs.microsoft.com/en-us/cpp/build/reference/zc-externconstexpr?view=vs-2017

note: 
> The /permissive- option does not enable /Zc:externConstexpr.

Reply:
I think this is fine, it's not like we're embedding huge amounts of data. I wish [std::embed](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2018/p1040r0.html) was a thing though...